### DISCUSSION TITLE
Half movie recognized and failed download

### DISCUSSION THREAD

**Discussion by pawelglucklich** [aclap-dev/video-downloadhelper#808] [Help and Q&A]
### Your question

When trying to download any movie from [https://35mm.online/](https://35mm.online/) service. VDH detect it but timer is always shorter than whole video actually is and once download progress reaches end it stalls and never finish it.

Seems like VDH has problem with detecting/accessing second part of movie. Any chance on looking into it? Unfortunately there isnt any other known to me source for those movies (they are old Polish movies and this site is government art repository).

### Addon & CoApp version and details

version: 9.0.2.3
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"474f35af1a114bdb"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "D:\\Download",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "yt_warning": false,
    "use_wide_ui": false,
    "use_sidebar": false,
    "open_count_store": 41,
    "successfull_dl": 2
}

**Comment by Mattmjs** [comment]
This site looks to use DRM which means everything is protected. Do you see this icon:

![DRM icon](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/b7f0f583-4372-4803-8744-141ca0e569ef)

**Comment by pawelglucklich** [comment]
thats seems to be the case, and i would imagine there is no solution to
this?

**Reply by Mattmjs** [reply]
DRM protects content, you end up with a useless file that has no sound or video. For your site I got a few minutes on a
download. Nothing played in my media player, just a black screen. If your desperate for this content then investigate
screen recording software.
